src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -1239,30 +1240,32 @@ bool UsgsAstroFrameSensorModel::setFocalPlane(double dx,double dy,
 
   double x;
   double y;
-  double fx;
-  double fy;
   double Jxx;
   double Jxy;
   double Jyx;
   double Jyy;
+  std::tuple<double, double> distortedPoint;
 
   // Initial guess at the root
```

Comment:
@jessemapel Is the majority of the setFocalPlane function removing distortion from dx, and dy?

Reply:
@acpaquette I'm not Jesse, but that's all it looks like it's doing.